content/docs/command-reference/config.md

```diff
@@ -238,6 +238,23 @@ to learn more about the state file (database) that is used for optimization.
   When an entry in the database is used (e.g. during the `dvc status`), DVC
   updates the timestamp on that entry. This way, when the database needs a
   cleanup, DVC can sort entries chronologically, and remove the oldest ones.
+  
+- `state.dir` - specify a custom location for the state file. This is
+  necessary when using DVC on NFS or other mounted volumes.
+  (By default, store SQLite-based remote indexes and state's
+  `links` and `md5s` caches in the repository itself to avoid any
+  possible state corruption in 'shared cache dir' scenario.) 
+  Use this override with care.
```

Comment:
What's the risk of overriding this? Maybe the warning should be more specific/scary 👻

Reply:
What should be avoided is that all of a sudden multiple users that each have the project checked out locally are using the same directory for the state.dir and index.dir. 
Let's assume we have a linux machine with multiple users and that each user has the project checked out locally at `/home/username/projectname`. No problems so far, because every folder is in fact different.
But if on that machine the users would decide to override the default behavior and put the `links` and/or `md5s` to `/tmp`, then we would end up with the same folder for state and index dir being used by multiple users. This would likely result in state curruptions.

Do you know what I mean? (but I figured such a scenario is overkill to describe here)... What do you think?

Reply:
Thanks for the explanation! I think we can summarize it like this:

```suggestion
  ⚠️ Make sure no other project shares the same `state.dir` location.
```

And same for `.dvc/tmp/index`, I guess?